tcmur_cmd_handler.c

```diff
@@ -747,6 +865,9 @@ static int tcmur_cmd_handler(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 	case WRITE_VERIFY_16:
 		ret = handle_write_verify(dev, cmd);
 		break;
+	case WRITE_SAME_16:
```

Comment:
@bgly 
I have manuall add the WRITE_SAME 0x41 (10) and WRITE_SAME (32) support just to do the test, and for the current code, tested using the sg_utils, the WRITE_SAME 0x41 (10) could work:

[root@localhost ~]# sg_write_same -S --in=/dev/zero --lba=10 --num=100 /dev/sdb --verbos
[root@localhost ~]# sg_write_same -R --in=/dev/zero --lba=10 --num=100 /dev/sdb --verbos

But the WRITE_SAME_32 0x0d (32) maybe need some extra code.

[root@localhost ~]# sg_write_same -T --in=/dev/zero --lba=10 --num=100 /dev/sdb --verbos
Write same: pass through os error: Invalid argument
Write same(32) command failed
[root@localhost ~]#